.github/workflows/lint.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   FORCE_COLOR: 2
-  NODE: 18
+  NODE: 21
```

Comment:
JFYI. For linting, using `latest` or `node` may be better, instead of a specific Node.js version.
See https://github.com/actions/setup-node#supported-version-syntax

Reply:
Hmm..  

Wouldn't that mean that the CI could break when a new Node.js version with breaking changes gets released and the project is not yet compatible with it?